functions/src/stats.js

```diff
@@ -1,3 +1,5 @@
+// TODO: Rename this file to stats.ts (or stats/index.ts) + refactor to make proper use of TypeScript
```

Comment:
[TODO] Rename this file to `stats.ts` (or `stats/index.ts`) + refactor to make proper use of TypeScript